manifests/repos.pp

```diff
@@ -1,5 +1,5 @@
 # Set up a repository for foreman
-define foreman::install::repos(
```

Comment:
That'd work fine I think.  We would just issue a new release of foreman_proxy that depends on the newer foreman module and get rid of it here in the next major.